sources/packages/web/src/views/aest/institution/ViewOfferingChangeRequest.vue

```diff
@@ -136,11 +136,7 @@ export default defineComponent({
           );
 
           router.push({
-            name: AESTRoutesConst.OFFERING_CHANGE_REQUEST_COMPLETE,
-            params: {
-              programId: props.programId,
-              offeringId: props.offeringId,
-            },
+            name: AESTRoutesConst.REQUEST_CHANGE_OFFERINGS,
```

Comment:
Can we also remove the obsolete route objects

route const `AESTRoutesConst.OFFERING_CHANGE_REQUEST_COMPLETE` 

`AppRoutes.ViewOfferingChangeRequestComplete`

and the route itself.
![image](https://github.com/bcgov/SIMS/assets/54600590/17073f22-6736-452f-97d8-b85a0644fdf6)

Reply:
I have deleted this now.